sdk/keyvault/.gitignore

```diff
@@ -2,6 +2,7 @@
 *.cer
 *.key
 *.pfx
+!azure-security-keyvault-jca/src/test/resources/certificate-util/**/*.pfx
```

Comment:
Hi, @g2vinay, @vcolin7 .
Could you please help to review this line? Just review this line is enough, not necessary to review the whole PR. And this line add test files in **.gitignore**. Without your approvement, I cannot merge this PR. 

Refs:
![image](https://github.com/user-attachments/assets/2618d02b-df16-47fd-927d-427ebf9450fc)

Reply:
While this works, I can't help but wonder if it would be better to have a separate `.gitignore` file in the `/keyvault` or `/azure-security-keyvault-jca` subdirectories to avoid adding library-specific stuff to the main one.

Any thoughts on this @srnagar, @alzimmermsft?

Reply:
I see .pfx files in `src/test/resources/certificate-util/` checked in. Looks like this is not ignoring these files. Also, do we need these private keys checked in?

Reply:
HI, @vcolin7 , @srnagar . 
Thanks for the comment.

1. These private keys are necessary for test.
2. In current PR, the modification is in `/sdk/keyvault/.gitignore`, not the `/.gitignore`.
3. I tried to use `/sdk/keyvault/azure-security-keyvault-jca/.gitignore`, ~but it didn't work.~ Now it works. I remember it didn't work when I do this last time. Maybe I made some mistake last time.
> ![image](https://github.com/user-attachments/assets/261471ee-0665-4b78-9d9e-b9607ad11fde)
4. Now I'll move it to `/sdk/keyvault/azure-security-keyvault-jca/.gitignore`